### PR TITLE
added and tested os.getcwd and os.path.abspath

### PR DESCRIPTION
**Pull request by GodefroyClair**
added the system calling function os.getcwd and os.path.abspath with some tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +185,8 @@ async def test_stuff():
   [#175](https://github.com/Tinche/aiofiles/issues/175)
 - Remove spurious items from `aiofiles.os.__all__` when running on Windows.
 - Switch to more modern async idioms: Remove types.coroutine and make AiofilesContextManager an awaitable instead a coroutine.
+- Added `aiofiles.os.path.abspath` and `aiofiles.os.getcwd`.
```
Comment (Tinche) [inline README.md:188]:
```suggestion
- Add `aiofiles.os.path.abspath` and `aiofiles.os.getcwd`.
```

**Review by Tinche — APPROVED** [review]

**Comment by Tinche** [comment]
Thanks!

**Comment by levincarmel** [comment]
@Tinche is there an intention to publish a new release with this commit? Thanks!

**Comment by Tinche** [comment]
> @Tinche is there an intention to publish a new release with this commit? Thanks!

Just released!

**Comment by levincarmel** [comment]
@Tinche thanks! what about updated types-aiofiles? I see mypy failures after upgrading

**Comment by Tinche** [comment]
> @Tinche thanks! what about updated types-aiofiles? I see mypy failures after upgrading

Different repo, different maintainers ;) See [typeshed.](https://github.com/python/typeshed/tree/main/stubs/aiofiles)

**Pull request merged**